core/src/test/java/com/linecorp/armeria/internal/server/annotation/AnnotatedValueResolverTest.java

```diff
@@ -429,6 +472,43 @@ void redundant2(@Param @Default("defaultValue") String var1) {}
         @Get("/r3/:var1")
         void redundant3(@Param Optional<String> var1) {}
 
+        @Get("/attribute-test")
+        void attributeTest(
+
+                @Attribute(value = "successPrefixOtherValuesOfOtherTypeInt", prefix = AttrDummyService.class)
+                int successPrefixOtherValuesOfOtherTypeInt,
+                @Attribute(value = "failPrefixNoneValuesOfOtherTypeInt")
```

Comment:
`value` is the default and can be omitted, ditto for the rest

```suggestion
                @Attribute("failPrefixNoneValuesOfOtherTypeInt")
```

Reply:
i fixed them all.